erpnext/accounts/doctype/bank_transaction/bank_transaction.py

```diff
@@ -1,15 +1,30 @@
 # Copyright (c) 2019, Frappe Technologies Pvt. Ltd. and contributors
 # For license information, please see license.txt
 
+import math
+
 import frappe
-from frappe.utils import flt
+from frappe.utils import cint, flt
 
 from erpnext.controllers.status_updater import StatusUpdater
 
 
 class BankTransaction(StatusUpdater):
+	_currency_precision = None
```

Comment:
This will be a class variable (?) and kinda behave weirdly in multi-tenant deploys.

Reply:
@casesolved-co-uk the override will continue to stay there when a request from another site is fired. 

It's basically a bug like this: https://github.com/frappe/frappe/pull/20052



```
worker starts: class attr is None
first request is fired: class attr is set to 2
second request is fired from some other site: class attr still stays 2 and site's preferences aren't respected.
```